restaurant/views.py

```diff
@@ -0,0 +1,218 @@
+from datetime import datetime, time, timedelta
+
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.core.exceptions import PermissionDenied
+from django.shortcuts import redirect, render
+from django.urls import reverse, reverse_lazy
+from django.utils import timezone
+from django.utils.dateparse import parse_date
+from django.utils.decorators import method_decorator
+from django.views import View
+from django.views.decorators.cache import cache_page
+from django.views.generic import DeleteView, DetailView, ListView, UpdateView
+
+from restaurant.forms import ReservationAdminForm, ReservationForm
+from restaurant.models import Reservation, Table, Worker
+from restaurant.services import round_time_to_next_slot, send_reservation_notification
+
+from .tasks import send_mail_feedback
+
+
+def home(request):
+    message_sent = False
+    if request.method == "POST":
+        email = request.POST.get("email")
+        message = request.POST.get("message")
+
+        send_mail_feedback.delay(email, message)
+
+        message_sent = True
+
+    return render(request, "restaurant/home.html", {"message_sent": message_sent})
+
+
+@cache_page(60 * 30)
+def history(request):
+    return render(request, "restaurant/history.html")
+
+
+@cache_page(60 * 10)
+def reservation_success(request):
+    return render(request, "restaurant/reservation_success.html")
+
+
+class ReservationView(View):
+    template_name = "restaurant/reservation.html"
+
+    def get(self, request):
+        tables = Table.objects.all().order_by("number")
```

Comment:
Можно сортировать чуть по-проще
```suggestion
        tables = Table.objects.order_by("number")
```